components/PeerReviews/PeerReviewInviteModal.tsx

```diff
@@ -0,0 +1,139 @@
+import { ReactElement, useRef, useState } from "react";
+import { css, StyleSheet } from "aphrodite";
+import { PeerReviewRequest } from "~/config/types/peerReview";
+import BaseModal from "~/components/Modals/BaseModal";
+import FormInput from "~/components/Form/FormInput";
+import Button from "~/components/Form/Button";
+import { inviteReviewer } from "./config/PeerReviewInviteAPI";
+import { MessageActions } from "~/redux/message";
+import { connect } from "react-redux";
+import Loader from "~/components/Loader/Loader";
+import colors from "~/config/themes/colors";
+
+type Props = {
+  peerReviewRequest: PeerReviewRequest;
+  fetchReviewRequests: Function;
+  isOpen: Boolean;
+  closeModal: Function;
+  setMessage: Function;
+  showMessage: Function;
+};
+
+function PeerReviewRequestModal({
+  peerReviewRequest,
+  fetchReviewRequests,
+  isOpen,
+  closeModal,
+  setMessage,
+  showMessage,
+}: Props): ReactElement {
+  const formInputRef = useRef();
+  const [email, setEmail] = useState('');
+  const [isLoading, setIsLoading] = useState(false);
+
+  const handleKeyDown = (e) => {
+    if (e?.key === 13 /*Enter*/) {
+      invite();
+    }
+  };
+
+  const handleSubmit = async (e) => {
+    e && e.preventDefault();
```

Comment:
nullchaining is much cleaner than nullchecking in most cases :) 
```suggestion
     e?.preventDefault();
```

Reply:
totally. that's my fault for copy/pasting